<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddOrEditConnectionDialog</class>
 <widget class="QDialog" name="AddOrEditConnectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>691</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Add/Edit Connection</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="commonWedget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>*Host</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="hostEdit">
          <property name="text">
           <string>127.0.0.1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Username</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="usernameEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>*Port</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="portEdit">
          <property name="text">
           <string>6379</string>
          </property>
          <property name="clearButtonEnabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="passwordEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_16">
        <item>
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>Connection name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="connectionNameEdit">
          <property name="text">
           <string>127.0.0.1:6379</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QCheckBox" name="sshCheckBox">
          <property name="text">
           <string>SSH</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="sslCheckBox">
          <property name="text">
           <string>SSL</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="sentinelCheckBox">
          <property name="text">
           <string>Sentinel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="clusterCheckBox">
          <property name="text">
           <string>Cluster</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="readonlyCheckBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Readonly</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QWidget" name="sshWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>SSH Config</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>288</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>*Host</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="sshHostEdit">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Username</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="sshUsernameEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>*Port</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="sshPortEdit">
          <property name="text">
           <string>22</string>
          </property>
          <property name="clearButtonEnabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="sshPasswordEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Private Key</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="sshPrivateKeyLineEdit"/>
        </item>
        <item>
         <widget class="QToolButton" name="sshPrivateKeyToolButton">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Passphrase</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="sshPassphraseEdit">
          <property name="text">
           <string/>
          </property>
          <property name="clearButtonEnabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Timeout</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="sshTimeoutEdit">
          <property name="text">
           <string>30</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QWidget" name="sslWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>SSL Config</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>288</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Private Key</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="sslPrivateKeylineEdit"/>
        </item>
        <item>
         <widget class="QToolButton" name="sslPrivateKeyToolButton">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Public Key</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="sslPublicKeylLineEdit"/>
        </item>
        <item>
         <widget class="QToolButton" name="sslPublicKeylToolButton">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>Authority</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="sslAuthorityLineEdit"/>
        </item>
        <item>
         <widget class="QToolButton" name="sslAuthorityToolButton">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QWidget" name="sentinelWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>Sentinel Config</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>288</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_14">
        <item>
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>*Master Group Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="sentinelMasterGroupNameEdit">
          <property name="text">
           <string>mymaster</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_15">
        <item>
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>Redis Node Password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="sentinelRedisNodePasswordEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddOrEditConnectionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>331</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
